clang/lib/Driver/Action.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include "clang/Driver/Action.h"
 #include "llvm/Support/ErrorHandling.h"
+#include "llvm/Support/raw_ostream.h"
```

Comment:
Is this needed?

Reply:
looks like those two fixes got forgotten - will re-implement.